linux/deploy_vm/rebuild_ubuntu_unattend_install_iso.yml

```diff
@@ -25,26 +25,26 @@
   register: extract_bios_cfg
 
 - name: "Get Release file path from md5sum.txt"
-  ansible.builtin.shell: "grep '/dists/.*/Release$' '{{ src_iso_dir_path }}/md5sum.txt' | awk '{print $NF}' | sed 's#^\\./##'"
+  ansible.builtin.shell: "grep -e '/dists/[^/]*/Release$' '{{ src_iso_dir_path }}/md5sum.txt' | awk '{print $NF}' | sed 's#^\\./##'"
   register: release_file_path_result
   ignore_errors: true
 
 - name: "Check Release file path"
   ansible.builtin.assert:
     that:
-      - release_file_path_result.stdout is defined
-      - release_file_path_result.stdout
+      - release_file_path_result.stdout_lines is defined
+      - release_file_path_result.stdout_lines | length > 0
     fail_msg: "Not found Release file path in md5sum.txt from the ISO image"
 
 - name: "Extract Release file from ISO"
   community.general.iso_extract:
     image: "{{ src_iso_file_path }}"
     dest: "{{ src_iso_dir_path }}"
     files:
-      - "{{ release_file_path_result.stdout }}"
+      - "{{ release_file_path_result.stdout_lines[0] }}"
 
 - name: "Get architectures from Release file"
-  ansible.builtin.shell: "grep '^Architectures:' {{ src_iso_dir_path }}/Release | awk '{print $NF}'"
+  ansible.builtin.shell: "grep '^Architecture' {{ src_iso_dir_path }}/Release | awk '{print $NF}'"
```

Comment:
update to `awk -F ':' '{print $NF}' | sed 's/^ *//'`.

Reply:
Thanks. Change it.